Content.Server/StationEvents/Events/Theta/ShipEvent-FlagCapture.cs

```diff
@@ -0,0 +1,122 @@
+using System.Threading;
+using Content.Server.GameTicking;
+using Content.Server.GameTicking.Rules.Components;
+using Content.Server.Theta.DebrisGeneration;
+using Content.Server.Theta.ShipEvent.Components;
+using Content.Server.Theta.ShipEvent.Systems;
+using Content.Shared.Roles.Theta;
+using Robust.Server.GameObjects;
+using Robust.Shared.Map;
+using Robust.Shared.Prototypes;
+using Timer = Robust.Shared.Timing.Timer;
+
+namespace Content.Server.StationEvents.Events.Theta;
+
+
+[RegisterComponent, Access(typeof(SEFCRule))]
+public sealed partial class SEFCRuleComponent : Component { } //yes partial is needed here
+
+[RegisterComponent]
+public sealed partial class SEFCFlagComponent : Component
+{
+    /// <summary>
+    /// Last team which has captured this flag
+    /// </summary>
+    public ShipEventFaction LastTeam = default!;
+}
+
+//Ship event flag capture
+public sealed class SEFCRule : StationEventSystem<SEFCRuleComponent>
+{
+    [Dependency] private TransformSystem _formSys = default!;
+    [Dependency] private DebrisGenerationSystem _debrisSys = default!;
+    [Dependency] private ShipEventFactionSystem _shipSys = default!;
+
+    private const string FlagPrototypeId = "SEFCFlag";
+    private const int PointsPerFlag = (int)10E6;
```

Comment:
```suggestion
    private const int PointsPerFlag = 10_000_000;
```
Мне кажется, так будет яснее. Не надо в уме представлять количество нулей. Ну и, а зачем давать 10 лямов очков сразу то..  1 лям же интереснее?

Reply:
ну можно просто 1 поставить, и понятно, и лям ровно